cmd/kube-apiserver/app/server.go

```diff
@@ -172,7 +172,9 @@ func CreateServerChain(runOptions *options.ServerRunOptions, stopCh <-chan struc
 	}
 	aggregatorConfig.ProxyTransport = proxyTransport
 	aggregatorConfig.ServiceResolver = serviceResolver
-	aggregatorServer, err := createAggregatorServer(aggregatorConfig, kubeAPIServer.GenericAPIServer, apiExtensionsServer.Informers)
+	aggregatorServer, err := createAggregatorServer(aggregatorConfig,
+		[]genericapiserver.DelegationTarget{kubeAPIServer.GenericAPIServer, apiExtensionsServer.GenericAPIServer},
```

Comment:
Why not just make the delegate of an DelegationTarget available with `Delegate() DelegationTarget`? I don't like to have the chain here another time, after having it implicitly in the construction of each apiserver. I think the `Delegate()` method would give you exactly the same.